source/config/__init__.py

```diff
@@ -914,6 +762,21 @@ def saveProfileTriggers(self):
 		self.triggersToProfiles.parent.write()
 		log.info("Profile triggers saved")
 
+	def getConfigValidationParameter(self, keyPath, validationParameter):
+		"""Get a config validation parameter
+		This can be used to get the min, max, default, or other values for a config key.
+		@param keyPath: a sequence of the identifiers leading to the config key. EG ("braille", "messageTimeout")
+		@param validationParameter: the parameter to return the value for. EG "max"
+		@type validationParameter: string
+		"""
+		if not keyPath or len(keyPath) < 1:
+			raise ValueError("Key path not provided")
+
+		spec = conf.spec
+		for nextKey in keyPath:
+			spec = spec[nextKey]
+		return conf.validator._parse_with_caching(spec)[2][validationParameter]

```

Comment:
The issue shown in the links to the NVDA-devel list were caused by a merge error (my fault). A line was missed from the config spec.

In order to get the validation parameters from a `[[__many__]]` section the specific name of the section must be used. As you would if you were to get a config value from that section. For an example of this see [Line 511 in source/gui/settingsDialogs.py](https://github.com/nvaccess/nvda/pull/6558/files/820c02fb95784fb20215dc86e67f540d068ae602#diff-597828d33ba584f58afddcc26b80a992R511) which looks like: 
`min=int(config.conf.getConfigValidationParameter(["speech", getSynth().name, "capPitchChange"], "min"))`